packages/headless/src/controllers/commerce/context/headless-context.ts

```diff
@@ -71,6 +84,7 @@ export interface ContextState {
   country: string;
   currency: CurrencyCodeISO4217;
   view: View;
+  user?: Location;
```

Comment:
I think we might as well go all the way and rename the controller state property.

Reply:
Definitely missed that one! Fixed in https://github.com/coveo/ui-kit/pull/4572/commits/79860a06299f788b1a37bf99e65c790ae19ed2aa